internal/flink/command_detached_savepoint_create.go

```diff
@@ -0,0 +1,79 @@
+package flink
+
+import (
+	"github.com/spf13/cobra"
+
+	cmfsdk "github.com/confluentinc/cmf-sdk-go/v1"
+
+	pcmd "github.com/confluentinc/cli/v4/pkg/cmd"
+	"github.com/confluentinc/cli/v4/pkg/examples"
+	"github.com/confluentinc/cli/v4/pkg/output"
+)
+
+func (c *command) newDetachedSavepointCreateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create [name]",
+		Short: "Create a Flink detached savepoint.",
+		Args:  cobra.ExactArgs(1),
```

Comment:
This is marked as `ExactArgs(1)` but the `Use` text uses `[ ]` instead of `< >`, which is our convention for optional arguments.